### DISCUSSION TITLE
Is Grafonnet working on Grafana V11.1

### DISCUSSION THREAD

**Discussion by DanickBG** [grafana/grafonnet#217] [General]
Hello all,

My team is planning to migrate Grafana from our current version 9 to version 11.1
I would like to ask in advance whether Grafonnet is supported on this higher version and whether there are some underwater rocks and hidden errors we should look for?

Thank you in advance!

**Comment by Duologic** [comment]
The model generally doesn't change that often, the v11 version should work well with v11.1, let us know if you are missing something.

**Reply by DanickBG** [reply]
The update went smoothly and there is nothing problematic! 
Thank you for the good work! :)
